chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java

```diff
@@ -1,40 +1,73 @@
 /*******************************************************************************
- * Copyright (c) 2014, 2018 Lablicate GmbH.
```

Comment:
Why did you remove the copyright header instead of extending it?

Reply:
I rewrote the whole file from scratch.

Reply:
Files shall inherit the header of the original, see comment from Wayne Beaton:
https://github.com/eclipse/chemclipse/issues/407

`"The short version is that when you split a file during a refactoring exercise, new files automatically inherit the header of the original. If even a fragment of a single line of the original content survives the refactoring (whether generated, rote, or otherwise), then the original copyright statement and author information should be retained."`